src/libs/web3/noconnector.ts

```diff
@@ -0,0 +1,34 @@
+import { Actions, Connector } from '@web3-react/types';
+
+export interface NoConnectorArgs {
+  actions: Actions;
+  name: string;
+  url: string;
+}
+
+export class NoConnectorError extends Error {
+  public constructor(name: string) {
+    super(`${name} not installed, redirecting to website...`);
+    this.name = NoConnectorError.name;
+    Object.setPrototypeOf(this, NoConnectorError.prototype);
+  }
+}
+
+export class NoConnector extends Connector {
+  /** {@inheritdoc Connector.provider} */
+  public declare readonly provider: undefined;
```

Comment:
I'm surprise you need to declare it. Can you explain why ?

Reply:
This was leftover from porting code from web3-react. Resolved